tests/test.py

```diff
@@ -128,6 +128,15 @@ def fetch_json(self, path, params={}, **kwargs):
 # TODO: split the handler tests into another file
 
 
+class TestRootHandler(BaseTestCase):
+    def test_home_page(self):
+        response = self.fetch('/')
+        files = open("index.html", "rb")
+        contents = files.read()
+        files.close()
+        self.assertEqual(response.body.decode('utf-8').split(), contents.decode('utf-8').split())
```

Comment:
Why are you using `.split()`?

Reply:
Actually index.html contains whitespaces so it was difficult to compare the raw data coming from response.body. Therefore I split them to make lists and then it can be easily compared. Is this something that shouldnt be used here?

Reply:
That is really trippy. Just verify that the response has the right header in it. That should suffice.

Reply:
Okay I can do that